README.md

```diff
@@ -39,6 +39,13 @@ or start on the [landing page](http://localhost:8000/)
 7. Create PR with your changes and attach it for validation on a platform
 
 
+Commands:
+1. Add ports and host name to settings.py
+2. Edit RUN python manage.py migrate -> Entrypoint command at the end
+3. Create Docker-compose.yml file and make a config
+4. Run docker compose up
+5. Check the result
+
```

Comment:
The format of the Readme should be like a user guide for someone, who has cloned your repository and is trying to run the project, not the steps you done in task.